components/browser/state/src/main/java/mozilla/components/browser/state/state/recover/RecoverableTab.kt

```diff
@@ -43,7 +46,9 @@ data class RecoverableTab(
     val createdAt: Long = 0,
     val lastMediaAccessState: LastMediaAccessState = LastMediaAccessState(),
     val private: Boolean = false,
-    val historyMetadata: HistoryMetadataKey? = null
+    val historyMetadata: HistoryMetadataKey? = null,
+    val source: Source = Source.Internal.None,
+    val restored: Boolean = true
```

Comment:
Do we ever set this to `false`? Since `RecoverableTab.toTabSessionState()` always calls `createTab()` with `restored = true` ... I wonder if that is needed and if yes why it's not used? :)

Reply:
Oh, good point - we don't actually need this!